src/components/Layout/Layout.js

```diff
@@ -40,6 +49,10 @@ const Content = styled.section`
   grid-area: content;
 `;
 
+const Temp = () => {
```

Comment:
I feel like we should remove the `Temp` component and it's route. Right now it's an example for an individual character route, but it's dead code and you have other examples for the actual links. Recreating this with something useful will be part of [Create nav for character](https://trello.com/c/hYqXgwNx/85-create-nav-for-character).

Thoughts?

Reply:
Agreed. Doesn't need to be there. I'll remove it.